bin/pic-create

```diff
@@ -79,6 +79,10 @@ if [ $# -eq 2 ] ; then
     src_path=$1
     dst_path=$2
     selected_default_folder_to_copy=$folder_to_clone
+    # Use example directory if no source directory available at current directory
+    if [[ -d $PIC_EXAMPLES/$src_path && ! -d ./$src_path ]]; then
```

Comment:
Can we please add here a check that if the directory exists as local and example directory and print a warning and say that the local directory is used.